mllib/pom.xml

```diff
@@ -109,7 +109,7 @@
     <dependency>
       <groupId>org.jpmml</groupId>
       <artifactId>pmml-model</artifactId>
-      <version>1.1.15</version>
+      <version>1.2.7</version>
```

Comment:
@vruusmann I've just realized this module is published as `pmml-model` and `jpmml-model`. Which is standard or preferred?